lib/vc.js

```diff
@@ -91,12 +92,20 @@ module.exports = {
 async function issue(options = {}) {
   const documentLoader = options.documentLoader || defaultDocumentLoader;
 
-  const {controller, domain} = options;
+  const {controller, domain, suite} = options;
   const purpose = options.purpose || new CredentialIssuancePurpose({
     controller,
     domain
   });
 
+  // check to make sure the `suite` and proof purpose have required params
+  if(!suite || !suite.verificationMethod) {
+    throw new TypeError('"suite.verificationMethod" property is required.')
+  }
+  if(purpose.term === 'assertionMethod' && !purpose.controller) {
+    throw new TypeError('"controller" property is required for proof purpose.')
+  }
```

Comment:
Do we need these additional checks... are they already covered via `CredentialIssuancePurpose`?

Reply:
@dlongley - I moved the '"controller" property is required' check into the constructor of `CredentialIssuancePurpose`.
I think the `verificationMethod` check is still needed, here? (Unless you think it should go somewhere else?)